src/applications/personalization/profile/actions/paymentInformation.js

```diff
@@ -43,30 +45,21 @@ export const EDU_PAYMENT_INFORMATION_SAVE_SUCCEEDED =
 export const EDU_PAYMENT_INFORMATION_SAVE_FAILED =
   'EDU_PAYMENT_INFORMATION_SAVE_FAILED';
 
-const captureDirectDepositErrorResponse = ({ error, apiEventName }) => {
-  const [firstError = {}] = error.errors ?? [];
-  const {
-    code = 'code-unknown',
-    title = 'title-unknown',
-    detail = 'detail-unknown',
-    status = 'status-unknown',
-  } = firstError;
-
-  captureError(error, {
-    eventName: apiEventName,
-    code,
-    title,
-    detail,
-    status,
-  });
-};
-
```

Comment:
This code I found to be hard to follow and introducing unneeded complexity.

In addition, when I looked at errors in sentry that used this call, they almost always resulted in the code, title, detail and status being 'unknown' so it wasn't getting the right things and resulting in no value for us.

 Error responses are instead now being captured in a more straightforward way for profile via sentry.